snapd.code-workspace

```diff
@@ -0,0 +1,52 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"go.testEnvVars": {"LANG": "C.UTF-8"},
+		"go.testFlags": ["-v","-check.v"]
+	},
+	"extensions": {
+		"recommendations": [
+			"golang.go", 
+			"766b.go-outliner", 
+			"jerrygoyal.shortcut-menu-bar",
```

Comment:
This one seems like a personal preference and is unrelated to Go or snapd.

Reply:
So I think we just disagree on expectations. I was expecting minimal support for using vscode with snapd. Here the Go extension is pretty much the only thing that is absolutely required and this is what I would expect to use.

We can leave recommendations and comments but the key difference is that the IDE won't prompt the (possibly inexperienced) user to install those.